src/components/molecules/WorldCard/WorldCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useCss } from "react-use";
+import { faCog, faSignInAlt } from "@fortawesome/free-solid-svg-icons";
+import classNames from "classnames";
+
+import { DEFAULT_VENUE_LOGO } from "settings";
+
+import { World } from "api/admin";
+
+import { WithId } from "utils/id";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import "./WorldCard.scss";
+
+export interface WorldCardProps {
+  world: WithId<World>;
+}
+
+export const WorldCard: React.FC<WorldCardProps> = ({ world }) => {
+  const cardVars = useCss({
+    backgroundImage: `url(${world.config.landingPageConfig.coverImageUrl})`,
+  });
+
+  const cardClasses = classNames("WorldCard", cardVars);
+
+  return (
+    <div className={cardClasses}>
+      <div className="WorldCard__info">
+        <div
+          className="WorldCard__logo"
+          style={{
+            backgroundImage: `url(${world.host?.icon ?? DEFAULT_VENUE_LOGO})`,
+          }}
+        />
+        <div className="WorldCard__titles">
+          <div className="WorldCard__world-name">{world.name}</div>
+          <div className="WorldCard__world-description">
+            {world.config.landingPageConfig.description}
+          </div>
+        </div>
+      </div>
+      <ButtonNG
+        variant="dark"
+        iconName={faSignInAlt}
+        className="WorldCard__button"
+      >
+        Enter dashboard
+      </ButtonNG>
+      <ButtonNG variant="dark" iconName={faCog} className="WorldCard__button">
+        Configure world
+      </ButtonNG>
```

Comment:
https://github.com/sparkletown/sparkle/blob/dda3d9dcc5620336aaac8c84f2de876fc319144a/src/utils/url.ts#L32

and

https://github.com/sparkletown/sparkle/blob/dda3d9dcc5620336aaac8c84f2de876fc319144a/src/utils/url.ts#L43

can be used to navigate to the world